proto/regen/ecocredit/v1/tx.proto

```diff
@@ -63,13 +62,13 @@ service Msg {
   // retired upon receipt.
   rpc Send(MsgSend) returns (MsgSendResponse);
 
-  // Retire retires a specified amount of tradable credits, removing the amount
```

Comment:
Why remove tradable? Only tradable credits can be retired.

Reply:
I think the added clarity is nice to have but maybe just because that's how I wrote it.